README.md

```diff
@@ -27,82 +28,112 @@ Install `yarn` globally:
 npm install --global yarn
 ```
 
-## Set environment variables
+## Add sf-toolkit package
+
+Create `package.json` by running the following command and run through the prompts:
+
+```
+yarn init
+```
+
+Install `sf-toolkit` as a dependency of your project:
 
-You can either add the environment variables locally in the file so you only need to add them once, or define them directly in the terminal. 
+```
+yarn add https://github.com/silverfin/sf-toolkit.git
+```
 
-### Local file 
+## Add scripts to packages.json
 
-Add a .env file in the root directory:
+You can add this `scripts` section to the `packages.json`, where we define `silverfin` as an alias to call our CLI.
 
 ```
-touch .env
+  "scripts": {
+    "silverfin": "node ./node_modules/sf_toolkit/bin/cli.js"
+  },
 ```
 
-Add the following variables in  .env:
+## Add your API credentials as environmental variables
+
+You could use a new `.env` local file, or add them to an existing file in your system (e.g. `~/.bash-profile` or `~/.bashrc`)
 
 ```
-SF_FIRM_ID="your firm ID"
-SF_ACCESS_TOKEN="your access token"
-NODE_ENV="development"
+export SF_API_CLIENT_ID=...
+export SF_API_SECRET=...
 ```
 
-### Linux / Mac terminal
+If you are only using one firm, you could set it up as an environmental variable. That way, you won't need to pass the `--firm <firmid>` option every time you run a command.
 
 ```
-export SF_FIRM_ID=<firm_id>
-export SF_ACCESS_TOKEN=<access_token>
+export SF_FIRM_ID=...
 ```
 
-### Windows terminal
+In case you need to use a different host, you can also set it up as an environmental variable. By default it would be `https://live.getsilverfin.com`.
 
 ```
-set SF_FIRM_ID=<firm_id>
-set SF_ACCESS_TOKEN=<access_token>
+export SF_HOST=...
 ```
 
-## Add sf-toolkit package
+# How to use it
 
-Create `package.json` by running the following command and run through the prompts:
+## Authorize the CLI
 
 ```
-yarn init
+yarn silverfin authorize
 ```
 
-Install `sf-toolkit` as a dependency of your project:
+## Import an existing reconciliation
 
 ```
-yarn add https://github.com/GetSilverfin/sf-toolkit.git
+yarn silverfin import-reconciliation --firm <firm-id> --handle <handle>
 ```
 
-## Create script shortcuts
+## Update an existing reconciliation
 
-Inside the `package.json`, add a new `scripts` block:
+```
+yarn silverfin update-reconciliation --firm <firm-id> --handle <handle>
+```
+
+## Import all existing reconciliations
 
 ```
-"scripts": {
-    "new-recon": "./node_modules/sf_toolkit/bin/cli.js new",
-    "import-recon": "./node_modules/sf_toolkit/bin/cli.js import",
-    "update-recon": "./node_modules/sf_toolkit/bin/cli.js persistReconciliationText"
-  }
+yarn silverfin import-all-reconciliations --firm <firm-id> --handle <handle>
```

Comment:
What's the difference between this and the import-reconciliation command, they look like they take the same set of arguments. Does the latter import the first reconciliation it finds in the firm with that handle and this one imports all reconciliations in a firm with a given handle?

Reply:
Indeed, `import-all-reconciliations` is not developed yet, just assigned the command name as a placeholder. But it should import all the reconciliations existing in the firm. The intention is to be used when you are setting up your local repository, and you still have no reconciliations in it. Similar to `import-all-shared-parts`. But this shouldn't have the `--handle <handle>` argument, only `--firm <firmid>`